web/src/components/network/WifiConnectionForm.tsx

```diff
@@ -162,7 +162,8 @@ export default function WifiConnectionForm({ network }: { network: WifiNetwork }
                 <FormSelectOption
                   key={security.value}
                   value={security.value}
-                  label={security.label}
+                  /* eslint-disable agama-i18n/string-literals */
+                  label={_(security.label)}
```

Comment:
This is good: an instance where the `_(expression)` is in the same file as the matching `N_("literal")`